src/assets/scss/components/event.scss

```diff
@@ -0,0 +1,131 @@
+$event-icon-palette: (
+    message: #2274d5,
+    inactive: #919191,
+    default: #57a01f
+    ) !default;
+$event-palette: (
+    primary: #57a01f,
+    active: #404040,
+    inactive: #919191,
+    notification: #d10910,
+    background: #fff,
+    background-inactive: #f8f8f8
+) !default;
+
+$black-shadow: rgba(0, 0, 0, .5);
+
+.footer {
+    text-align: center;
+}
+
+.content {
+    margin-left: auto;
+    margin-right: auto;
+    padding: inherit;
+    white-space: nowrap;
+    @include breakpoint(medium) {
+        width: 80%;
+    }
+}
+
+.icon {
+    background-color: map-get($map: $event-icon-palette, $key: inactive);
+    border: 2px;
```

Comment:
jinak pořadí vlastností, to je sice taková hříčka, ale je nepsané pravidlo, dávat společné vlastnosti k sobě tedy v tomto případě border, šířku, výšku respektive dib a vat... Ještě by se dalo říct, že pozice a zobrazení jsou důležité, tak se to dává nahoru a tak, ale to už je  můj názor..

Reply:
stahl jsem si Sass linter a upravil css do jednotneho stylu, taky mi prijde lepsi davat vlastnosti k sobe. jeste nemam vyvynuty zadny specificky style zapisu, tak mi to prijde o neco lepsi nez to pripisovat nahodne jako predtim :)